pkg/cmd/cobra/config.go

```diff
@@ -261,30 +261,25 @@ func (c *LogConfig) flags() []string {
 
 	// level
 	if c.Level != "" {
-		flags = append(flags, c.Level)
+		flags = append(flags, fmt.Sprintf("level=%s", c.Level))
```

Comment:
Is it possible to import flags here, enabling to use the string constants that you added?